addons/skin.confluence/720p/DialogKeyboard.xml

```diff
@@ -20,6 +20,54 @@
 				<param name="CloseButtonLeft" value="770" />
 				<param name="CloseButtonNav" value="3" />
 			</include>
+			<control type="group">
+				<left>0</left>
+				<top>-90</top>
+				<visible>Control.IsVisible(313)</visible>
```

Comment:
Being nitpicky: I dont really like group controls which toggle visibility based on a control which is part of the group itself. It would be more clear if 313 would be outside of this group.